glom/core.py

```diff
@@ -1832,11 +1845,13 @@ def glom(target, spec, **kwargs):
 
 def _glom(target, spec, scope):
     parent = scope
-    scope = scope.new_child()
-    parent[LAST_CHILD_SCOPE] = scope
-    scope[T] = target
-    scope[Spec] = spec
-    scope[UP] = parent
+    scope = scope.new_child({
+        T: target,
+        Spec: spec,
+        UP: parent,
+        CHILD_ERRORS: [],
+    })
```

Comment:
This reads better, nice.

Reply:
nice, I mostly did it actually to unpack the function a little -- I felt bad about adding even more stuff to _glom() and played with timeit, discovered that the __setitem__ calls on the ChainMap completely dominate the list construction time

building the dict constant like this and passing it in is much, much faster than the other way -- like 5-10x as fast

Reply:
totally makes sense, I was thinking the same, but didn't wanna prematurely optimize :)